web/blog/public-roadmap-and-open-rfcs/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Public Roadmap and Open RFCs
+description: Open Source needs Open Governance
+authors: mavam
+draft: true
+date: 2022-09-07
+tags: [roadmap, github, open-source]
+---
+
+We are happy to announce that we have published [our technical roadmap][roadmap]
```

Comment:
```suggestion
We are happy to announce that we have published [our engineering roadmap][roadmap]
```

I think engineering roadmap is the more commonly used term for this, and since it's also the one we use internally it makes sense to use it here.